src/features/events/components/SelectionBarEllipsis.tsx

```diff
@@ -100,6 +99,7 @@ const SelectionBarEllipsis = () => {
       onSelect: () => {
         model.updateEvents(selectedEventIds, true, false);
       },
+      textColor: '',
```

Comment:
IMO a better solution to this would have been to add a type to the array on line 50, so that Typescript would know that not all objects have to look exactly the same (i.e. that the `textColor` property is optional).